### PR TITLE
Expanding bronze: Armor

### PR DESCRIPTION
**Pull request by Jarewill**
#### Summary
Content "Adds more bronze armor"

#### Purpose of change
Follows up on #55815
Currently the only craftable armors from bronze are historical Greek and Egyptian pieces that require a recipe book to even craft.
Meaning in runs where recipe books aren't that common, bronze was rather useless apart the generic bronze greaves item.
This is supposed to fix it by adding a survivor-grade bronze armor, which is inferior to the historical pieces.

#### Describe the solution
Added a bronze cuirass that is thinner, covers less and is balanced awkwardly, giving more encumbrance.
Added a bronze helmet that follows the same rules.
Added bronze vambraces that cover only the forearms, similar to the bronze greaves.

#### Describe alternatives you've considered
Pretty much all stats are in consideration to be changed.
Alternatively not including some/all pieces at all.

#### Testing
Created a debug character and spawned in necessary materials.
Crafted and wore the armors

#### Additional context
Posting as a draft, because I am really uncertain of my choices here.
Unlike editing existing items, adding brand new ones causes me anxiety, as I really don't know if my choices are correct or not.
I will appreciate input on this.

### CODE REVIEW

File: data/json/items/armor/arms_armor.json
Comment (bombasticSlacks) [inline data/json/items/armor/arms_armor.json:907]: greaves also cover the knees if you wanted to keep parity.

File: data/json/items/armor/torso_armor.json
Comment (bombasticSlacks) [inline data/json/items/armor/torso_armor.json:42]: I would say this item could be a smidge better if it requires all the same proficiencies and books other armors do, and 10hrs to construct.

It's less thick and covering than the bell cuirass so perhaps its encumbrance could be dropped somewhat to compensate.

**Review by bombasticSlacks — APPROVED** [review]

File: data/json/items/armor/arms_armor.json
Comment (Jarewill) [reply data/json/items/armor/arms_armor.json:907]: When I searched up images of vambraces, most didn't have elbow protection, which is why I left it out.
I can update the vambraces to cover the elbows as well if that's wanted.

File: data/json/items/armor/torso_armor.json
Comment (Jarewill) [reply data/json/items/armor/torso_armor.json:42]: Hmmm, alright, I can adjust the item and the recipe a bit.
The bell cuirass currently has 7 encumbrace when fitted, so I could make this item have 6, as I don't want to make it too low.
And I could also shave off another hour from the recipe, as it's currently only an hour shorter than the bell.
Do you think that's a good way to go about it?

File: data/json/items/armor/arms_armor.json
Comment (bombasticSlacks) [reply data/json/items/armor/arms_armor.json:907]: nah if they don't they don't, I wouldn't worry about that.

File: data/json/items/armor/torso_armor.json
Comment (bombasticSlacks) [reply data/json/items/armor/torso_armor.json:42]: I didn't realize the bell could be fitted... I guess my point is, it's not like this item is a poorly made armor. It takes hours, fabrication, book references, and forge tools so if its got worse coverage, worse protection and worse encumbrance over the bell cuirass, whats the point? I don't think it's imbalanced, or wrong as is but I don't know if it's really filling any niche.

Comment (Jarewill) [reply data/json/items/armor/torso_armor.json:42]: I planned for it to be a basic piece of armor for survivors that can't find an armorsmithing book, because the recipe is auto-learned at fabrication 5.
The book references are there to give the player the recipe for it if their skill is at 4, but it should be auto-learned at 5.
That gives bronze use in runs with no access to books, or at least that's what I was planning it to do.

Though I do agree the recipe should be tweaked, I am just unsure how exactly.
As is, it's basically just copy-pasted recipe for the bell cuirass with a few tweaks, such as taking slightly less bronze to craft.

Comment (bombasticSlacks) [reply data/json/items/armor/torso_armor.json:42]: ohhhh I didn't see the autolearn, my bad. In that case I think this is good as is 👍 

might be worth adding a sentence to the description saying something like "this isn't particularly well designed" so players can at a glance see that they would want the bell cuirass if they have the books.

**Comment by Jarewill** [comment]
Seeing as #55971 got merged and it also modifies torso_armor.json, will there be conflicts between those two PRs?
I wouldn't want to accidentally revert the updates made in that PR.
If so, what should I do to fix the issues? Merge the updated file into my branch?

**Comment by bombasticSlacks** [comment]
If you don't touch the lamelar cuiras directly you should be fine Github is smart enough to handle changes to seperate parts of the same file. Though someone will need to restart your tests they failed for unrelated reasons.

**Comment by bombasticSlacks** [comment]
Yeah it all looks good, you could do an empty commit in git to refire the tests, things I think are stable now. Without "basic build passed" as one of the GitHub tags this will end up overlooked for a while. To get it the first major test suite needs to pass. 

You can do an empty commit by:

git commit --allow-empty -m "refire tests"

**Comment by Jarewill** [comment]
Do you happen to know how to do an empty commit using GitHub's web editor?
I can't seem to figure out how to do it exactly.

**Comment by bombasticSlacks** [comment]
I don't know I'll ask for someone to kick on the discord

**Comment by Jarewill** [comment]
Thanks!

**Comment by Jarewill** [comment]
There's one failed test, but the PR seems to have the BasicBuildPassed label, so I assume everything's fine now?
In any case, thank you both for getting this working!

**Comment by bombasticSlacks** [comment]
yeah that failure isn't your fault, it's an odd one tho

**Comment by LyleSY** [comment]
Kicked. Looks like you do have some conflicts to resolve here before it can be merged

**Comment by LyleSY** [comment]
The conflicts were pretty easy for me to understand, this should be good to merge after the tests finish

**Review by LyleSY — APPROVED** [review]

**Comment by Jarewill** [comment]
How should I resolve those conflicts?
Or were they already resolved by you?
Sorry, I'm still new to github.

**Comment by LyleSY** [comment]
Standard etiquette would be to let you fix it, but I got the sense you needed some help and/or I am an insufferable busybody and I want to see this get merged because it's fun and I want to build stuff based on it so I did it for you. You're good here.

File: data/json/items/armor/arms_armor.json
Comment (Maleclypse) [inline data/json/items/armor/arms_armor.json:906]:
```suggestion
```

**Review by LyleSY — APPROVED** [review]

**Pull request merged**
